packages/account-v2/src/constants/constants.ts

```diff
@@ -1,12 +1,12 @@
-export const LANDING_COMPANY = {
+export const landingCompany = {
```

Comment:
planning to move constants to camel case as well?
I think the team decided to keep the case for constants

Reply:
Will check with the team on this 👌